Mutagen.Bethesda.Analyzers.Testing/TestModule.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.IO.Abstractions;
+using Autofac;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Mutagen.Bethesda.Analyzers.Autofac;
+
+namespace Mutagen.Bethesda.Analyzers.Testing
+{
+    public class TestModule : Module
```

Comment:
Same with this

Reply:
Migrating over `TestModule` would mean `Mutagen.Bethesda.Analyzers.TestingUtils` would need to import `AutoFac`, which shouldn't be enforced on anyone downstream.

Autofac is used in the engine library, but that's not meant to be published and imported by people